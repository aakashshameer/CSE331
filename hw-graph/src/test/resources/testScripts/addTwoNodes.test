#Test to create two nodes in the graph

#Create a graph
CreateGraph dgraph

#Add a node
AddNode dgraph a
AddNode dgraph b

#Print nodes in the graph
ListNode dgraph

#Print outgoing edges from a
ListChildren dgraph a

#Print outgoing edges from b
ListChildren dgraph b