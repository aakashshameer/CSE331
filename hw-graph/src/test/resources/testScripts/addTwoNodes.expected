#Test to create two nodes in the graph

#Create a graph
created graph dgraph

#Add a node
added node a to dgraph
added node b to dgraph

#Print nodes in the graph
dgraph contains: a b

#Print outgoing edges from a
the children of a in dgraph are:

#Print outgoing edges from b
the children of b in dgraph are: